daycare/models.py

```diff
@@ -39,3 +49,26 @@ def create(email, username, password, name, description, price_per_day, capacity
         new_daycare.save()
 
         return new_daycare
+
+
+class Image(models.Model):
+    url = models.CharField(max_length=1000)
```

Comment:
any reason not to use the url field?

Reply:
I see. you can use a custom validator to allow this usage, but it defies the purpose.